.travis/Containerfile.ci_base

```diff
@@ -0,0 +1,82 @@
+FROM fedora:31
```

Comment:
I wonder why not pre-build a container and upload it to quay, to avoid building it each time from `fedora` image?

Reply:
That's on our to-do list.

Reply:
I broke it out into this extra Containerfile to support just this. I wanted to postpone it until we have proof from a few plugins that we do not need to change it every day.
There are few more things to consider though:
- Where do we want to host this Containerfile?
- Can plugins (or branches) access older/alternative versions of that image?
- How can we incorporate testing against python 3.6?